index.js

```diff
@@ -44,16 +46,22 @@ module.exports = {
           replacement: "import lodash from 'lodash'\nconst isPlainObject = lodash.isPlainObject"
         }
       ]
-    })
+    });
+
+    reduxTree = esTranspiler(reduxTree, {
```

Comment:
@rwjblue ah this looks like it did the trick

```js
    let addon = this.addons.find(addon => addon.name === 'ember-cli-babel');
    reduxTree = addon.transpileTree(reduxTree, {
      babel: {
        plugins: ['babel-plugin-transform-object-rest-spread']
      }
    });
```

I finally read some documentation on the subject ;)

https://libraries.io/npm/ember-cli-babel